opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -12,7 +12,214 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+from collections import OrderedDict
+from collections import deque
+from collections import namedtuple
+import threading
+import typing
+
 from opentelemetry import trace as trace_api
+from opentelemetry.sdk import util
+
+try:
+    from collections.abc import MutableMapping
+    from collections.abc import Sequence
+except ImportError:
+    from collections import MutableMapping
+    from collections import Sequence
+
+
+MAX_NUM_ATTRIBUTES = 32
```

Comment:
These might move to the exporters, see https://github.com/open-telemetry/opentelemetry-specification/issues/182.